Configurations/unix-Makefile.tmpl

```diff
@@ -298,6 +298,7 @@ OPENSSLDIR={- #
 LIBDIR={- our $libdir = $config{libdir};
           unless ($libdir) {
               $libdir = "lib$target{multilib}";
+              $libdir =~ s/\s+//g;
```

Comment:
My only concern for 3.0 is this change.  The rest should be confined to non-stop only.

I can't see why this would break anything...

Reply:
> My only concern for 3.0 is this change.  The rest should be confined to non-stop only.
> 
> I can't see why this would break anything...

My thought as well. However, multiple multilibs are currently broken. I can split the PR if you want.

Reply:
Let's see what the OTC think on the call tomorrow.